returnn/training.py

```diff
@@ -573,3 +573,179 @@ def hash(cls, kwargs):
         }
 
         return super().hash(d)
+
+
+class GetBestEpochJob(Job):
+    """
+    Provided a RETURNN model directory and a score key, finds the best epoch.
+    The sorting is lower=better, so to acces the model with the highest values use negative index values (e.g. -1 for
+    the model with the highest score)
+
```

Comment:
I would maybe add that "score" in RETURNN terminology actually (usually) refers to a loss, such as CE or FER.